### PR TITLE
I18n support

### PR DESCRIPTION
**Pull request by abhishek-kaushik**
Added Support for translations and updated vulnerable packages.

### CODE REVIEW

File: inc/namespace.php [hunk 1]
```diff
@@ -35,6 +35,8 @@ function enqueue_block_editor_assets() {
 		'restBase' => esc_url_raw( get_rest_url() ),
 		'postSelectEndpoint' => '/rbmh/v1/post-select',
 	] );
+
+	wp_set_script_translations( 'hm-gb-tools-editor', 'rbmh' );
```
Comment (mattheu) [inline inc/namespace.php]: `rbmh` is the text domain from the project this was built for, we should change all instances of rbmh to something else. e.g. `hm-gb-tools`?

File: js/post-select/containers/term-select-form-field.js [hunk 2]
```diff
@@ -89,13 +89,15 @@ class TermSelect extends React.Component {
 	}
 
 	render() {
+		const labelText = sprintf( __( 'Filter by %s' ), this.props.label );
```
Comment (mattheu) [inline js/post-select/containers/term-select-form-field.js]: missing textdomain for translation

**Comment by abhishek-kaushik** [comment]
I have changed text-domain to `hm-gb-tools`. but strings inside `js/post-select/containers` are not being translated.

**Comment by mattheu** [comment]
I'm not sure i'm afraid - I can't see why this is not working!

**Comment by stuartshields** [comment]
I'll have a look today

**Comment by stuartshields** [comment]
@mattheu So I tracked the issue down to the following error:
```
react Class properties must be methods
```

I've let @abhishek-kaushik know the issue and he's going to go and update accordingly.

File: js/post-select/containers/current-selection.js [hunk 3]
```diff
@@ -7,11 +7,14 @@ import { fetchPostsById } from '../utils/fetch';
...
 
 class CurrentSelectionContainer extends Component {
-	state = {
-		posts: [],
-		isLoading: false,
-		isMounted: false,
-	};
+	constructor( props ) {
+		super( props );
+		this.state = {
```
Comment (stuartshields) [inline js/post-select/containers/current-selection.js]: @abhishek-kaushik just being picky, add a blank line above.

**Pull request merged**